recipe/conda_build_config.yaml

```
@@ -1,2 +1,11 @@
+osx_impl:  # [osx]
+  - mps    # [osx]
+  - nomps  # [osx]
+
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - "12.3"                 # [osx]
+MACOSX_SDK_VERSION:        # [osx]
+  - "12.3"                 # [osx]
+
```

```suggestion
MACOSX_DEPLOYMENT_TARGET:  # [osx]
  - 11.0                   # [osx and arm64]
  - 10.9                   # [osx and x86_64]
  - 12.3                   # [osx]
MACOSX_SDK_VERSION:        # [osx]
  - 11.0                   # [osx and arm64]
  - 10.9                   # [osx and x86_64]
  - 12.3                   # [osx]

zip_keys:                           # [osx]
  - - osx_impl                      # [osx]
    - MACOSX_DEPLOYMENT_TARGET      # [osx]
    - MACOSX_SDK_VERSION            # [osx]
```

can you try with this and rerender